pom.xml

```diff
@@ -42,7 +42,7 @@
             <dependency>
                 <groupId>io.jenkins.tools.bom</groupId>
                 <artifactId>bom-2.401.x</artifactId>
-                <version>2102.v854b_fec19c92</version>
+                <version>2133.v2e6c00fe4d61</version>
```

Comment:
This update is required so we can use the changes introduced in https://github.com/jenkinsci/branch-api-plugin/pull/374 which allow using the display name from the `ObjectMetadataAction` defined for the `SCMHead` associated with the job.

Reply:
Sorry, I should have been more explicit with my comment. This was so we could utilize the `MultiBranchProjectDisplayNamingTrait` that was added to the `branch-api-plugin` which is one way for users to have the PR title be displayed as the job name.

Reply:
We're not referencing it directly ourselves but by upgrading the `branch-api-plugin` to a version that has it, the trait is "automatically" made available as an available option (as shown in one of the screenshots above).

Reply:
Yeah, that's actually a good point. This was mostly to make it more convenient for the user to have the option readily available as part of this feature. However, I've just had a closer look at the bom version itself and it seems that there are some breaking changes included with such an upgrade. With that said, I think it's best to revert this change and as you mentioned the user can always install a newer version of the branch-api-plugin if they require it.